packages/components/src/SubHeaderBar/InputTitleSubHeader/InputTitleSubHeader.component.js

```diff
@@ -1,50 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import classNames from 'classnames';
-import Skeleton from '../../Skeleton';
-import TitleSubHeader from './TitleSubHeader.component';
-import InlineFormSubHeader from './InlineFormSubHeader.component';
-import theme from './InputTitleSubHeader.scss';
-import Icon from '../../Icon';
-
-function InputTitleSubHeader({ title, iconId, editMode, loading, inProgress, ...rest }) {
-	if (loading) {
-		return <Skeleton type={Skeleton.TYPES.text} size={Skeleton.SIZES.large} />;
-	}
-	return (
-		<div
-			className={classNames(theme['tc-subheader-details'], 'tc-subheader-details', {
-				[theme['tc-subheader-details-blink']]: inProgress,
-				'tc-subheader-details-blink': inProgress,
-			})}
-		>
-			{iconId && (
-				<Icon
-					name={iconId}
-					className={classNames(theme['tc-subheader-details-icon'], 'tc-subheader-details-icon')}
-				/>
-			)}
-			{editMode ? (
-				<InlineFormSubHeader title={title} {...rest} />
-			) : (
-				<TitleSubHeader title={title} disabled={inProgress} {...rest} />
```

Comment:
![image](https://user-images.githubusercontent.com/13314851/43524238-0a6357ee-959f-11e8-8620-db4c69fe42ad.png)
You should consider also this case

Reply:
✅